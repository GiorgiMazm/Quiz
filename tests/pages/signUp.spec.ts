import { expect } from "vitest";
import { setup, $fetch } from "@nuxt/test-utils";

describe.skip("example", async () => {
  await setup({
    server: true,
    port: 3000,
  });
  it("Renders Hello Nuxt", async () => {
    const page = await $fetch("/");
    expect(page).toMatchSnapshot();
  });
});
