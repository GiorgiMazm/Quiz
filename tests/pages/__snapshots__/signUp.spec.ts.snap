// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`example > Renders Hello Nuxt 1`] = `
"<!DOCTYPE html>
<html >
<head><meta charset=\\"utf-8\\">
<meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1\\"><link rel=\\"preload\\" as=\\"style\\" href=\\"/_nuxt/@fs/Users/giorgi/Desktop/Projects/Quiz/node_modules/tailwindcss/tailwind.css\\"><link rel=\\"preload\\" as=\\"style\\" href=\\"/_nuxt/Users/giorgi/Desktop/Projects/Quiz/node_modules/tailwindcss/tailwind.css\\"><link rel=\\"modulepreload\\" as=\\"script\\" crossorigin href=\\"/_nuxt/Users/giorgi/Desktop/Projects/Quiz/node_modules/nuxt/dist/app/entry.js\\"><link rel=\\"stylesheet\\" href=\\"/_nuxt/@fs/Users/giorgi/Desktop/Projects/Quiz/node_modules/tailwindcss/tailwind.css\\"><link rel=\\"stylesheet\\" href=\\"/_nuxt/Users/giorgi/Desktop/Projects/Quiz/node_modules/tailwindcss/tailwind.css\\"></head>
<body ><div id=\\"__nuxt\\"><div><div><header class=\\"bg-sky-600 h-[10vh] flex items-center\\"><div class=\\"container mx-auto\\"><nav class=\\"flex justify-between\\"><ul class=\\"flex w-1/5 justify-between\\"><li class=\\"cursor-pointer hover:text-cyan-500\\"><a aria-current=\\"page\\" href=\\"/\\" class=\\"router-link-active router-link-exact-active\\">Home777</a></li><li class=\\"cursor-pointer hover:text-cyan-500\\"><a href=\\"/quizzes\\" class=\\"\\">Quizzes</a></li><!----></ul><div><a href=\\"/signIn\\" class=\\"cursor-pointer hover:text-cyan-500\\">Sign in </a><!----></div></nav></div></header></div><div class=\\"min-h-[80vh]\\"><div class=\\"mx-auto container text-center flex h-[80vh] items-center\\"><div class=\\"justify-self-center\\"><h1 class=\\"font-bold text-6xl text-cyan-500 pt-9\\"> This is home page! </h1><p class=\\"text-3xl mt-5 text-cyan-400\\"> Welcome to the Quiz App! Test your knowledge and have fun with our challenging quizzes. Choose from a variety of exciting topics and see how well you can score, or create your own quiz for others and share the link with them! Are you ready to take on the challenge? Let&#39;s get started and show off your expertise! </p><a href=\\"/quizzes\\" class=\\"inline-block py-8 px-12 mt-5 bg-sky-600 rounded-xl text-3xl hover:bg-red-600\\">List of quizzes</a></div></div></div><div><footer class=\\"bg-sky-800 h-[10vh] flex items-center text-center capitalize\\"><div class=\\"container mx-auto\\">footer</div></footer></div></div></div><script type=\\"application/json\\" id=\\"__NUXT_DATA__\\" data-ssr=\\"true\\">[[\\"Reactive\\",1],{\\"data\\":2,\\"state\\":4,\\"_errors\\":8,\\"serverRendered\\":9},{\\"HAum8Lt9X7\\":3},\\"\\",{\\"$sauth:data\\":5,\\"$sauth:lastRefreshedAt\\":6,\\"$sauth:loading\\":7},null,[\\"Date\\",\\"2023-07-28T17:24:21.038Z\\"],false,{\\"HAum8Lt9X7\\":5},true]</script><script>window.__NUXT__={};window.__NUXT__.config={public:{auth:{computed:{origin:\\"\\",pathname:\\"/api/auth\\",fullBaseUrl:\\"/api/auth\\"},isEnabled:true,session:{enableRefreshPeriodically:false,enableRefreshOnWindowFocus:true},globalAppMiddleware:{isEnabled:false,allow404WithoutAuth:true,addDefaultCallbackUrl:true},provider:{type:\\"authjs\\",trustHost:false,defaultProvider:\\"\\",addDefaultCallbackUrl:true}}},app:{baseURL:\\"/\\",buildAssetsDir:\\"/_nuxt/\\",cdnURL:\\"\\"}}</script><script type=\\"module\\" src=\\"/_nuxt/@vite/client\\" crossorigin></script><script type=\\"module\\" src=\\"/_nuxt/Users/giorgi/Desktop/Projects/Quiz/node_modules/nuxt/dist/app/entry.js\\" crossorigin></script></body>
</html>"
`;
